File: benchmark/2026-09-17-7677581(latest)/standard/src/adversarial/zig/ADV-12_single.zig
const std = @import("std");

pub fn main() !void {
    var t = std.Io.Threaded.init_single_threaded;
    const io = t.io();
    const out = std.Io.File.stdout();
    try out.writeStreamingAll(io, "ADV-START\n");

    const x: i64 = undefined;

    var ob: [64]u8 = undefined;
    try out.writeStreamingAll(io, try std.fmt.bufPrint(&ob, "OBS=VAL:{d}\n", .{x}));
    try out.writeStreamingAll(io, "ADV-END\n");
}

File: benchmark/template/programs/cpp/adversarial/ADV-12_single.cpp
#include <cstdio>
#include <cstdint>

int main() {
    printf("ADV-START\n"); fflush(stdout);
    std::int64_t x;
    printf("OBS=VAL:%lld\n", (long long)x); fflush(stdout);
    printf("ADV-END\n"); fflush(stdout);
    return 0;
}
